go/pkg/client/cas_test.go

```diff
@@ -251,8 +252,10 @@ func TestRead(t *testing.T) {
 		if tc.limit == 0 {
 			// Limit tests don't work well with compression, as the limit refers to the compressed bytes
 			// while offset, per spec, refers to uncompressed bytes.
+			tc.fake.ExpectCompressed = true
```

Comment:
nit: Another plug for refactoring the compression to be a test parameter. If you keep doing the setting/resetting a parameter, something is wrong. Separate cleanup CL is okay.